files/build_templates/init_cfg.json.j2

```diff
@@ -36,7 +36,6 @@
                    ("database", "always_enabled", false, "always_enabled"),
                    ("lldp", "enabled", true, "enabled"),
                    ("pmon", "enabled", true, "enabled"),
-                   ("pmon", "enabled", false, "enabled"),
```

Comment:
@Junchao-Mellanox why NOT remove ("pmon", "enabled", true, "enabled")? and keep it same as previous 
```
        "pmon": {
            "state": "enabled",
            "has_timer": false,
            "has_global_scope": true,
            "has_per_asic_scope": "False",
            "auto_restart": "enabled",
            "high_mem_alert": "disabled"
        },
```

Reply:
We will use a timer to start pmon in fastreboot/warmreboot flow. By doing this, we would save fast reboot traffic down time.